pySPT/preAnalysis/expDisplacement_noGUI.py

```diff
@@ -0,0 +1,140 @@
+"""
+@author: Johanna Rahm
+Research group Heilemann
+Institute for Physical and Theoretical Chemistry, Goethe University Frankfurt a.M.
+
+Determine how far a particle moves between frames in the x/y-plane [nm].
```

Comment:
adjust docstring description (add your name under authors, why was this file created, what is the difference to expDisplacement.py)